commands/plugin_e2e_test.go

```diff
@@ -5,14 +5,84 @@ package commands
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/mattermost/mattermost-server/v5/model"
+
+	"path/filepath"
+
 	"github.com/spf13/cobra"
 
 	"github.com/mattermost/mmctl/client"
 	"github.com/mattermost/mmctl/printer"
 )
 
+func (s *MmctlE2ETestSuite) TestPluginAddCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	mmPath := os.Getenv("MM_SERVER_PATH")
+	pluginPath := filepath.Join(mmPath, "../mmctl/commands/test_files/testplugin.tar.gz")
+	fmt.Println(pluginPath)
```

Comment:
We can directly use the `testplugin` file that exists in the server repository and not add it to `mmctl`:

```suggestion
	pluginPath := filepath.Join(os.Getenv("MM_SERVER_PATH"), "tests", "testplugin.tar.gz")
```